src/store/modules/user.store.js

```diff
@@ -25,6 +50,27 @@ export const userStore = {
         },
         getAllFailure(state, error) {
             state.all = { error };
+        },
+        settingRequest(state, user){
+            state.status = { loggingIn: true };
+            state.user = user;
+        },
+        loginSuccess(state, user) {
+            state.status = { loggedIn: true };
+            state.user = user;
+        },
+        loginFailure(state) {
+            state.status = {};
+            state.user = null;
+        },
+        refreshUser(state, error){
+            state.all = { error};
+            let user=localStorage.getItem("user");
```

Comment:
in one line you are using `"` and on the next line you are using `'` ? Please don't code like a school kid anymore.